rasa/core/constants.py

```diff
@@ -17,6 +17,7 @@
 DEFAULT_LOCK_LIFETIME = 60  # in seconds
 
 REQUESTED_SLOT = "requested_slot"
+SHOULD_NOT_BE_SET = "should_not_be_set"
```

Comment:
Let's add a comment how this constant is used?
I also think this is a bit more explicit
```suggestion
SLOT_SHOULD_NOT_BE_SET = "should_not_be_set"
```

Reply:
it's not only slot

Reply:
ok, then `STATE_SHOULD_NOT_BE_SET`?